src/plugins/saved_objects_management/public/management_section/objects_table/components/table.tsx

```diff
@@ -298,7 +303,7 @@ export class Table extends PureComponent<TableProps, TableState> {
           {
             name: i18n.translate(
               'savedObjectsManagement.objectsTable.table.columnActions.viewRelationshipsActionName',
-              { defaultMessage: 'Relationships' }
+              { defaultMessage: 'View object relationships' }
```

Comment:
Please keep the default message as it was because it may break cypress test.

Reply:
Done